package.json

```diff
@@ -8,12 +8,23 @@
     "clean": "rimraf ./build",
     "deploy": "firebase deploy",
     "dev": "webpack-dev-server --hot --silent",
+    "lint": "npm run format -- --check",
+    "format": "prettier-standard 'src/**/*.{js,jsx}' 'src/**/*.{css,md,json}'",
+    "precommit": "lint-staged && npm test",
     "start": "firebase emulators:start",
     "start:client": "firebase emulators:start --only hosting",
     "storybook": "start-storybook -c .storybook",
     "test": "jest --silent --coverage",
     "test:ci": "jest --silent"
   },
+  "lint-staged": {
+    "src/**/*.{css,md,json}": [
+      "prettier-standard"
+    ],
+    "src/**/*.{js,jsx}": [
+      "prettier-standard"
+    ]
```

Comment:
Can we group these as they do the same thing?

Reply:
For sure, updated!